components/table/index.jsx

```diff
@@ -109,23 +111,36 @@ class InfiniteTable extends React.PureComponent {
   }
 
   handleRowClick = event => {
-    const { expandable } = this.props
-    if (!expandable) return
+    if (event.detail === 1) {
```

Comment:
I do not see any way this `1` comes here and what it means. Detail technically should not but meaningfully should be an object.

Reply:
This is a number of times element is clicked. See [docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event). It does work very well but it's good to have it there.

Reply:
Thank you. I did not know that 🙂